### DISCUSSION TITLE
How do objects of different species merge?

### DISCUSSION THREAD

**Discussion by bill0919** [stuart-lab/signac#767] [Q&A] [answered]
I have human and chimpanzee data. And I created a Signac object for each of them. But I wonder how to combine them together. Or can you tell me how to compare TFs activity of the same type of cells from different species?

**Comment by timoast** [comment, marked as answer]
To merge objects containing data from different species, you will need to identify equivalent / orthologous peaks across the species, and rename the peaks so that they match across the two objects.

To compare TF activity within a cell type across species, one approach would be to annotate cells consistently across the two objects, and the compare the chromVAR motif deviation scores in the same cell types.

**Reply by bill0919** [reply]
I found the method of genomic coordinate transformation from version 1.2 of vignettes (https://satijalab.org/signac/1.2.0/articles/integration.html). Can I perform genome coordinate transformation first and then compare the chromVAR Motif deviation scores in the same cell types? Is it impossible to add fragments file using this approach?

**Reply by timoast** [reply]
Yes, you could lift-over peak coordinates between different species and then rename the peaks in one of the objects.

If your goal is to compare chromVAR motif deviations, I'm not sure that you need to transform peak coordinates. You could simply look at the same motif across the two different species.

You can still include the fragment file in your object

**Reply by bill0919** [reply]
I have a question.

The gene coordinates of chimpanzee counts matrix has been modified to the gene coordinates of human by LiftOver, while the fragment file is still the gene coordinates of  chimpanzee. And the gene annotation file in the object is EnsDb.Hsapiens.v75. In this case, can I run `GeneActivity`?

Because I found `FeatureMatrix` in `GeneActivity`, which requires reading fragment files, I had the above problems.

**Reply by timoast** [reply]
Right, your data was mapped to the genome for each species. All you've done is modify the peak matrix, the fragment files will still be mapped to each genome. I'd suggest computing gene activities and peak matrices using the original genome coordinates for each species, and if you want to then compare them you identify othologous genes or lift over peak coordinates

**Comment by pengxin2019** [comment]
> I have human and chimpanzee data. And I created a Signac object for each of them. But I wonder how to combine them together. Or can you tell me how to compare TFs activity of the same type of cells from different species?

Hi there,
I hope everything is going well with your research. 
I am also working on scATAC data of  non-human and non-mouse species.  I am wondering whether you have done **co-accessible networks using Cicero** or **per-cell motif activity computation by running chromVAR**. Are they doable for you?

Thanks
Best
Penny

**Reply by bill0919** [reply]
I did not run co-accessible networks.
